pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -76,55 +126,44 @@ type manager struct {
 	certAccessLock           sync.RWMutex
 	cert                     *tls.Certificate
 	shouldRotatePercent      uint
+	forceRotation            bool
 }
 
 // NewManager returns a new certificate manager. A certificate manager is
 // responsible for being the authoritative source of certificates in the
 // Kubelet and handling updates due to rotation.
-func NewManager(
-	certSigningRequestClient certificatesclient.CertificateSigningRequestInterface,
-	template *x509.CertificateRequest,
-	usages []certificates.KeyUsage,
-	certificateStore Store,
-	certRotationPercent uint) (Manager, error) {
-
-	cert, err := certificateStore.Current()
+func NewManager(config *Config) (Manager, error) {
+	cert, forceRotation, err := getCurrentCertificateOrBootstrap(
```

Comment:
Way don't we just call rotate inside this function instead of saving forceRotation?